Themes/default/scripts/smf_fileUpload.js

```diff
@@ -515,4 +516,15 @@ function smf_fileUpload(oOptions) {
 			myDropzone.emit("complete", mock);
 		});
 	}
+
+	function hide_file_progress_and_all_buttons_if_needed() {
```

Comment:
nitpick: can you change the function name to camelCase please?

Reply:
Done